examples/ota-provider-app/esp32/README.md

```diff
@@ -7,6 +7,18 @@ A prototype application that demonstrates OTA provider capabilities.
 -   This example supports ESP32 and ESP32C3. For details please check
     [here](https://github.com/shubhamdp/connectedhomeip/tree/shubhamdp-patch-1/examples/all-clusters-app/esp32#supported-devices).
```

Comment:
Can you please correct this link to https://github.com/project-chip/connectedhomeip/tree/master/examples/all-clusters-app/esp32#supported-devices

Reply:
Done